### PR TITLE
Filter Fairbnb protected email that cause blinking

### PR DESCRIPTION
**Pull request by martinratinaud**

### CODE REVIEW

File: declarations/Fairbnb.coop.json [hunk 1]
```diff
@@ -5,6 +5,9 @@
       "fetch": "https://fairbnb.coop/terms-of-use/",
       "select": [
         "#main-content"
+      ],
+      "filter": [
+        "filterProtectedEmail"
```
Comment (MattiSG) [inline declarations/Fairbnb.coop.json:10]: Naming a filter function `filter` seems a bit redundant, what about something like:
```suggestion
        "unifyEmailProtectedAddresses"
```

**Review by MattiSG — APPROVED** [review]

**Comment by MattiSG** [comment]
The test shows that this filter is not idempotent.

**Comment by MattiSG** [comment]
Why was this merged? Did you fix the failing test?

**Comment by martinratinaud** [comment]
@MattiSG What test do you talk about ? [Checks](https://github.com/OpenTermsArchive/p2b-compliance-declarations/pull/148/checks) seem to be ok.
Can you please elaborate?

**Pull request merged**